dataproc/create_cluster_json.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from google.cloud import dataproc_v1beta2
+import google.protobuf.json_format as json_format
+import google.cloud.dataproc_v1beta2.proto.clusters_pb2 as clusters
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "D:\\GCP\\xxxx.json"
```

Comment:
This should generally be configured outside of the sample, and I would advise removing this line.

Reply:
Removed the line